portfolio/src/main/webapp/script.js

```diff
@@ -136,6 +144,26 @@ function getLoginStatus() {
   return fetch(url).then((response) => response.json());
 }
 
+function getDateFromTimestamp(timestamp) {
+  const date = new Date(timestamp);
+
+  const day = formatTwoDigits(date.getDay());
+  const month = formatTwoDigits(date.getMonth());
+  const year = date.getFullYear();
+  const dateStr = `${day}.${month}.${year}`;
+
+  const hours = formatTwoDigits(date.getHours());
+  const minutes = formatTwoDigits(date.getMinutes());
+  const seconds = formatTwoDigits(date.getSeconds());
+  const timeStr = `${hours}:${minutes}:${seconds}`;
+
+  return timeStr + ' ' + dateStr;
+}
+
+function formatTwoDigits(toFormat) {
+  return ('0' + toFormat).substr(-2);
```

Comment:
Could be replaced with just ``` Date.toLocaleString ```:

```javascript 
new Date(2020, 10, 10, 9, 9).toLocaleString(); // will give you "10.11.2020 09:09:00"
```

It will format the timestamp according to current user locale settings. You could also
use a fixed locale to avoid using "MM/DD/YYYY" format. 

[toLocaleString](https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleString)